### PR TITLE
Beyza GOK week1 DATABASES

### PR DESCRIPTION
**Pull request by beyzagoknl**

### CODE REVIEW

File: Week1/homework/exercise1/queries.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        invited_by NVARCHAR(100));`,
+
+  `CREATE TABLE IF NOT EXISTS Room (
+        room_no INT NOT NULL PRIMARY KEY,
+        room_name VARCHAR(100) UNIQUE,
+        floor_number INT);`,
+
+  `CREATE TABLE IF NOT EXISTS Meeting (
+        meeting_no INT AUTO_INCREMENT PRIMARY KEY,
+        meeting_title VARCHAR(200) UNIQUE NOT NULL,
```
Comment (unmeshvrije) [inline Week1/homework/exercise1/queries.js:20]: Nice well indented queries and good thinking about constraints like NOT NULL and UNIQUE, 💪  Well done !

File: Week1/homework/exercise2/connection.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  password: "hyfpassword",
+  database: "new_world",
+});
+
+connection.connect();
+
+const selectQuery = (query) => {
+  connection.query(query, (err, result) => {
+    if (err) throw err;
+    console.table(result);
```
Comment (unmeshvrije) [inline Week1/homework/exercise2/connection.js:15]: One Nitpick: 
The following code is more readable
```
if (err) {
    throw err;
} else {
    console.log("some message")
}
```

**Review by unmeshvrije — APPROVED** [review]
Well done!

**Comment by beyzagoknl** [comment]
> Well done!

Great to hear that from you. Thank you your time and your comments.

**Pull request closed without merging**